Sources/ProjectSpec/Dependency.swift

```diff
@@ -17,12 +17,14 @@ public struct Dependency: Equatable {
     public var codeSign: Bool = true
     public var removeHeaders: Bool = true
     public var link: Bool = true
+    public var implicit: Bool?
```

Comment:
As a nil value has no real meaning here we can make it default to false. Something like `embed` is an optional as the default changes depending on other factors